roles/openshift_master/tasks/main.yml

```diff
@@ -164,6 +164,20 @@
     - restart master api
     - restart master controllers
 
+- name: Configure master to use etcd3 storage backend on 3.6 clean installs
+  yedit:
+    src: /etc/origin/master/master-config.yaml
+    key: kubernetesMasterConfig.apiServerArguments.storage-backend
+    value:
+      - etcd3
```

Comment:
@smarterclayton This is the only key we need to set? Some places it mentioned switching from json to protobuf but the etcd v3 migration document mentions only setting this key.